### PR TITLE
added seed data

### PR DESCRIPTION
**Pull request by dereklou**
added the .json with 100 puppies and a README file with all of the instructions of how to get this imported into the db

### CODE REVIEW

File: data/puppy_seed.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	{"src": "http://40.media.tumblr.com/4cdcfd8ce7e76b79b215c32b0510fb3c/tumblr_o2y7g8pKn31ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://s-media-cache-ak0.pinimg.com/564x/59/be/1e/59be1e6b950d67540412177131f63f31.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://s-media-cache-ak0.pinimg.com/564x/c1/e2/12/c1e212fb5552d947b6f07b9470f793d7.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://s-media-cache-ak0.pinimg.com/564x/04/e4/fb/04e4fb941625f694b12e79881cff91ba.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://wesleying.org/wp-content/uploads/2015/11/cute-dogs-01.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://mbtimetraveler.files.wordpress.com/2016/01/cute-dog-names-11.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://s3.amazonaws.com/assets.prod.vetstreet.com/64/5c/2914709f4bc38fcca4126dfa2c74/american-eskimo-dog-AP-GE1BT2-590sm52213.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0}
+]
```
Comment (naji247) [inline data/puppy_seed.json:99]: Dude, where is the new line.

#UseVIM

#SquashThis

#YouHaveToBecauseDarrenAlwaysMadeMeDoIt

Also it's ugly and if I open that file Vim will auto add a new line

File: data/README.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Use this command from DogShow/ to import the data from puppy_seed.json
+
+mongoimport --db dogshow --collection dogs --drop --file data/puppy_seed.json --type json --jsonArray
```
Comment (naji247) [inline data/README.txt:3]: Anyway we can have this run when we call "npm start" or "npm i"?

Comment (naji247) [inline data/README.txt:3]: Usually node apps are "npm i" then "npm start". We shouldn't deviate

Comment (dereklou) [inline data/README.txt:3]: depends on if you want it to reset all the scores and wins/loss counters everytime you do npm start

Comment (naji247) [inline data/README.txt:3]: Perhaps there's a way to do it after "npm i" so that it only seeds when we install

**Pull request merged**